thunder/tests/test_networks.py

```diff
@@ -445,7 +454,8 @@ def test_hf_for_nemo(model_id):
     # fullgraph=True used to work with transformers 4.45.2, but it doesn't work
     # with 4.46.2 because of re.findall usage in the loss function
     fullgraph = False
-    compiled_model = thunderfx(model, fullgraph=fullgraph)
+    # compiled_model = thunderfx(model, fullgraph=fullgraph)
+    compiled_model = thunder.jit(model, fullgraph=fullgraph)
```

Comment:
Wasn't this failing due to unsupported argument as I thought `thunder.jit` doesn't have `fullgraph` argument?
Or `**compile_options` takes keyword argument so there wouldn't be errors for unsupported args?

```suggestion
    compiled_model = thunder.jit(model, fullgraph=fullgraph)
```

Reply:
Nope. One can see the logs by clicking through the CI links below:

```
=========================== short test summary info ============================
FAILED thunder/tests/test_networks.py::test_hf_for_nemo[bigcode/starcoder2-7b] - AssertionError
FAILED thunder/tests/test_networks.py::test_hf_for_nemo[microsoft/Phi-3-mini-128k-instruct] - AssertionError: expected tensor with (48,), cuda:0, torch.float32, requires_grad=False, got (1,), cuda:0, torch.bfloat16, False
FAILED thunder/tests/test_networks.py::test_thunderfx_mistral_nemo_small - AssertionError
============ 3 failed, 29 passed, 172 warnings in 172.63s (0:02:52) ============
```

so I guess your theory is correct that we do not error out due to unsupported (kw)args.
I have a vague memory of us discussing doing that though; maybe it's just a warning?

Reply:
That is one of the things I dislike about the options.